rust/src/lib.rs

```diff
@@ -1,30 +1,30 @@
-pub mod seer {
+pub mod events {
```

Comment:
Are the changes in this file intended? Seems like something re-ordered things here.